crates/aptos/src/create_account/mod.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! A command to create a new account on-chain
+//!
+//! TODO: Examples
+//!
+
+use crate::{
+    common::types::{EncodingOptions, KeyInputOptions},
+    CliResult, Error as CommonError,
+};
+use anyhow::Error;
+use aptos_crypto::{ed25519::Ed25519PrivateKey, PrivateKey};
+use aptos_rest_client::Client as RestClient;
+use aptos_sdk::{
+    transaction_builder::TransactionFactory,
+    types::{chain_id::ChainId, transaction::authenticator::AuthenticationKey, LocalAccount},
+};
+use aptos_transaction_builder::aptos_stdlib;
+use aptos_types::account_address::AccountAddress;
+use clap::Parser;
+use reqwest;
+
+/// Command to create a new account on-chain
+///
+#[derive(Debug, Parser)]
+pub struct CreateAccount {
+    #[clap(flatten)]
+    key_input_options: KeyInputOptions,
+    #[clap(flatten)]
+    encoding_options: EncodingOptions,
+    /// Address of account you want to create
+    account: AccountAddress,
+    /// Chain ID
+    chain_id: u8,
+    #[clap(long, default_value = "https://fullnode.devnet.aptoslabs.com")]
+    node_url: String,
+}
+
+impl CreateAccount {
+    async fn get_account(
+        &self,
+        account: AccountAddress,
+    ) -> Result<serde_json::Value, reqwest::Error> {
+        reqwest::get(format!("{}/accounts/{}", self.node_url, account))
+            .await?
+            .json()
+            .await
+    }
+
+    async fn get_sequence_number(&self, account: AccountAddress) -> Result<u64, CommonError> {
+        let account_response = self
+            .get_account(account)
+            .await
+            .map_err(|err| CommonError::UnexpectedError(err.to_string()));
+        let sequence_number = &account_response?["sequence_number"];
+        match sequence_number.as_str() {
+            Some(number) => Ok(number.parse::<u64>().unwrap()),
+            None => Err(CommonError::UnexpectedError(
+                "Sequence number not found".to_string(),
+            )),
+        }
+    }
+
+    async fn post_account(
+        &self,
+        sender_key: Ed25519PrivateKey,
+        sender_account: AccountAddress,
+        sequence_number: u64,
+    ) -> Result<String, Error> {
+        let client = RestClient::new(reqwest::Url::parse(&self.node_url).unwrap());
```

Comment:
I *think* you can actually put this `parse` in the command line args, and it'll tell the errors if it fails before it even does anything (and in the standard CLI output)